helm-chart/renku/templates/data-service/deployment.yaml

```diff
@@ -96,6 +96,17 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.global.redis.existingSecret }}
                   key: {{ .Values.global.redis.existingSecretPasswordKey }}
+            - name: SOLR_URL
+              value: "http://{{ template "solr.fullname" . }}:{{ .Values.global.solr.port }}"
+            - name: SOLR_CORE
+              value: {{ index .Values.solr.coreNames 1 }}
```

Comment:
This will break when someone overrides the helm values with the wrong number of core names.

Reply:
Yes I know; or the order is changed etc. It's crazy. I did it analogous to what already exists (the other core was picked by a `first` template function…). This whole yaml programming stuff is a giant mess to me anyways 😅 The plan is to phase out one of them eventually.